lib/commands/exam.js

```diff
@@ -239,14 +241,26 @@ module.exports = TestCommand.extend({
     const testemEvents = this.testemEvents;
     const ui = this.ui;
     const events = config.custom_browser_socket_events || {};
-    const testExecutionFileName = `test-execution-${Date.now()}.json`;
-    const browserExitHandler = function() {
+
+    const browserExitHandler = function(failed = false) {
+      if (!failed && commands.get('loadBalance')) {
+        let browserId;
+        try {
```

Comment:
1. I think just because it's not able to find a browser id to log out it shouldn't stop executing the test..
2. Now that I think about it..hmm I guess there wouldn't be no cases of browserId not being available?
3. I will change to log out something like `A browser Id is not available. A browser is exiting..`